defaults/py_modules/steamback/__init__.py

```diff
@@ -334,60 +385,58 @@ def _rcf_is_valid(self, root_dir: str, rcf: list[str]):
-        d = self._get_gamedir(game_info["game_id"])
-        path = os.path.join(d, "remotecache.vdf")
+        dirs = self._get_gamedir(game_info["game_id"])
+        paths = [os.path.join(d, "remotecache.vdf") for d in dirs]
         # logger.debug(f'Read rcf {path}')
 
         rcf = []
-        if os.path.isfile(path):
-            with open(path) as f:
-                s = f.read()  # read full file as a string
-                lines = s.split('\n')
-                # logger.debug(f'file is {lines}')
-
-                # drop first two lines because they are "gameid" {
-                lines = lines[2:]
-                # We look for lines containing quotes and immediately preceeding lines with just an open brace
-                prevl = None
-
-                skipping = False
-                for l in lines:
-                    s = l.strip()
-                    if skipping:  # skip the contents of {} records
-                        if s == "}":
-                            skipping = False
-                    elif s == "{":
-                        if prevl:
-                            # prevl will have quote chars as first and last of string.  Remove them
-                            filename = (prevl[1:])[:-1]
-                            rcf.append(filename)
-                            prevl = None
-
-                        # Now skip until we get a close brace
-                        skipping = True
-                    else:
-                        prevl = s
-        else:
-            logger.debug(f'No rcf {path}')
-            return None
+        for path in paths:
+            if os.path.isfile(path):
+                with open(path) as f:
+                    s = f.read()  # read full file as a string
+                    lines = s.split("\n")
+                    # logger.debug(f'file is {lines}')
+
+                    # drop first two lines because they are "gameid" {
+                    lines = lines[2:]
+                    # We look for lines containing quotes and immediately preceeding lines with just an open brace
+                    prevl = None
+
+                    skipping = False
+                    for l in lines:
+                        s = l.strip()
+                        if skipping:  # skip the contents of {} records
+                            if s == "}":
+                                skipping = False
+                        elif s == "{":
+                            if prevl:
+                                # prevl will have quote chars as first and last of string.  Remove them
+                                filename = (prevl[1:])[:-1]
+                                rcf.append(filename)
+                                prevl = None
+
+                            # Now skip until we get a close brace
+                            skipping = True
+                        else:
+                            prevl = s
+            else:
+                logger.debug(f"No rcf {path}")
```

Comment:
Removed the return  None. as now it need to search  inside the for

Reply:
And the rcf will remain empty, so the rest of the logic won't find any files to backup